tests/kernel/queue/src/main.c

```diff
@@ -8,25 +8,16 @@
 #include "test_queue.h"
 
 #ifndef CONFIG_USERSPACE
-static void test_queue_supv_to_user(void)
-{
-	ztest_test_skip();
-}
-
-static void test_auto_free(void)
-{
-	ztest_test_skip();
-}
-
-static void test_queue_alloc_prepend_user(void)
-{
-	ztest_test_skip();
-}
-
-static void test_queue_alloc_append_user(void)
-{
-	ztest_test_skip();
-}
+#define dummy_test(_name)		\
+	static void _name(void)		\
+	{				\
+		ztest_test_skip();	\
+	}
+
+dummy_test(test_queue_supv_to_user);
+dummy_test(test_queue_alloc_prepend_user);
+dummy_test(test_queue_alloc_append_user);
+dummy_test(test_auto_free);
```

Comment:
Seems not very relevant to the patch.  Maybe consider splitting this out to a separate cleanup patch.